deploy.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
I think this is 644 (not executable) and might need to be changed, like with:

    chmod +ux deploy.sh

Reply:
Oops, yes. Now it's 755